app/services/quotes.js

```diff
@@ -0,0 +1,27 @@
+const axios = require('axios');
+const logger = require('../logger');
+
+const getQuote = async (author, genre, count) => {
+  const url = 'https://quote-garden.herokuapp.com/api/v3/quotes/random';
+  let quote = {};
+  const params = {
+    author,
+    genre,
+    count
+  };
+
+  try {
+    const resService = await axios.get(url, { params });
```

Comment:
Here if it seems good to you you can do a destructuring so that you do not have to do so much nesting later.